src/toil/batchSystems/kubernetes.py

```diff
@@ -1040,14 +1092,14 @@ def shutdown(self) -> None:
             for pod in ourPods:
                 try:
                     if pod.status.phase == 'Failed':
-                            logger.debug('Failed pod encountered at shutdown: %s', str(pod))
+                            logger.debug('Failed pod encountered at shutdown:\n%s', self._pretty_print(pod))
                     if pod.status.phase == 'Orphaned':
-                            logger.debug('Orphaned pod encountered at shutdown: %s', str(pod))
+                            logger.debug('Orphaned pod encountered at shutdown:\n%s', self._pretty_print(pod))
```

Comment:
Same with these, I think that they could be `info` level (though only for the failures/orphans).